hangman.py

```diff
@@ -0,0 +1,234 @@
+    """
+    Check if letter is word, else subtracts point
+    """
+    letters_guessed.add(letter)
+    if letter in secret_word:
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+    else:
+        if letter in LIST_VOWELS:
+            guesses_point -= 2
+        else:
+            guesses_point -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+    return letters_guessed, guesses_point
+
+
+def print_classic_warning(warnings_point, guesses_point, secret_word, letters_guessed):
+    global count_star
+    if count_star != 0:
+        warnings_point, guesses_point = warnings("you are playing classic hangman", warnings_point, guesses_point, secret_word, letters_guessed)
+    else:
+        print("Oops, you are playing classic hangman:", get_guessed_word(secret_word, letters_guessed))
+        count_star += 1
+    return warnings_point, guesses_point
+
+
+def check_letter(letter, warnings_point, guesses_point, secret_word, letters_guessed):
+    '''
+    This function check if letter is correct. So letter is correct, if it is acsii_letters, len(letter) is 1 and
+    letter not in letters_guessed. Additionally, it check that letter is or is not a GAME_WITH_HINTS("*")
+    '''
+    if letter == GAME_WITH_HINTS:
+        warnings_point, guesses_point = print_classic_warning(warnings_point, guesses_point, secret_word, letters_guessed)
+    elif letter not in string.ascii_letters or len(letter) != 1:
+        warnings_point, guesses_point = warnings("That is not a valid letter or you print more than one letter.", warnings_point, guesses_point, secret_word, letters_guessed)
+    elif letter in letters_guessed:
+        warnings_point, guesses_point = warnings("You have already guessed that letter:", warnings_point, guesses_point, secret_word, letters_guessed)
+    else:
+        letters_guessed, guesses_point = letter_in_word(secret_word, letter, guesses_point, letters_guessed)
+    return warnings_point, guesses_point
+
+
+def result_of_game(secret_word, letters_guessed, guesses_point):
+    if is_word_guessed(secret_word, letters_guessed) == 0:
+        final_points = guesses_point * len(set(secret_word))
+        print("------")
+        print("Congratulations, you won!")
+        print("Word:", secret_word)
+        print("Your total score for this game is:", final_points)
+    elif is_word_guessed(secret_word, letters_guessed) != 0 and guesses_point <= 0:
+        print("------")
+        print("Sorry, you ran out of guessed. The word was", secret_word)
+
+
+def hangman(secret_word, type_of_game):
+    '''
+    Game Hangman. "request" show hangman(classic) or hangman with hints.
+    Print information for user, start loop, check user choose hangman or hangman with hints,
+    Check input letter and call letter_in_word, check if user guessed word.
+    '''
+
+    guesses_point = 6
+    warnings_point = 3
```

Comment:
I believe it makes sense to create a constants for initial values of these variables